src/domain/session/room/timeline/tiles/BaseMessageTile.js

```diff
@@ -21,9 +21,11 @@ import {getIdentifierColorNumber, avatarInitials, getAvatarHttpUrl} from "../../
 export class BaseMessageTile extends SimpleTile {
     constructor(options) {
         super(options);
+        this._tilesCreator = options.tilesCreator;
```

Comment:
About this, I'd prefer to not add another member to this class if possible. Can we have a method where we create the replyTile that is called from the ctor and updateEntries when needed and receiving the tilesCreator as an argument?

Reply:
Turns out, we don't really need to create the reply tile initially. We can just create it on updateEntry since we'll be notified when context-entry is loaded.